### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by joshuaskelton**
Fixing spelling mistakes.

Fixes: #131

### CODE REVIEW

**Comment by MSLaguana** [comment]
Looks good

**Comment by digeff** [comment]
:+1: Is there any kind of dictionary check we can put as part of the build or tslint?

File: src/debugger/launcher.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class Launcher {
                             generator.step("mobilePlatform.enableJSDebuggingMode");
                             return mobilePlatform.enableJSDebuggingMode(runOptions);
                         }).then(() =>
-                            Log.logMessage("Debugging session started succesfuly."));
+                            Log.logMessage("Debugging session started successfully."));
```
Comment (meenakb) [inline src/debugger/launcher.ts]: You will get a merge error for this one as I just fixed this.

**Comment by meenakb** [comment]
:+1:

**Comment by dlebu** [comment]
LGTM!

File: src/debugger/launcher.ts (hunk 1 above)
Comment (joshuaskelton) [inline src/debugger/launcher.ts]: Wow. That was fast. :smile:

**Pull request merged**